scripts/install_plugin_modules.js

```diff
@@ -116,7 +116,8 @@ function addFolder (name, version) {
 }
 
 function folder (name, version) {
-  const basename = [name, version].filter(val => val).join('@')
+  const _name = name ? name.replace('/', '#') : name
```

Comment:
Could you add a comment that we're doing this to avoid needing to create the parent dir for scoped packages?